### ISSUE TITLE
azure vm create jump server issue 

### ISSUE THREAD

**Issue by piggyvenus** [VeerMuchandi/openshift-on-azure#2]
I will get an error to with the command "azure vm create --resource-group $resourceGroupName --name $vmName --location $location --vm-size $vmSize --subnet-id $subnetId --nic-names $nicName --os-type linux --image-urn RHEL --storage-account-name $storageAccountName --admin-username <username> --ssh-publickey-file ~/.ssh/id_rsa.pub.

I fixed by add "--disable-boot-diagnostics" to the end of command line. By default, The switch parameter to disable the boot diagnostics (by default enabled) for VM. It cause the creation to fail.

I was able to create vm with "--disable-boot-diagnostics" option disabled.

**Comment by kylebassett** [comment]
Yes I got stuck on this also - this is the command I used to make it work ok.. I missed the issue entry - thanks @piggyvenus 
azure vm create --resource-group $resourceGroupName     --name $vmName     --location $location     --vm-size $vmSize     --subnet-id $subnetId     --nic-names $nicName     --os-type linux     --image-urn RHEL     --storage-account-name $storageAccountName     --admin-username $adminUserName     --ssh-publickey-file ~/.ssh/id_rsa.pub --disable-boot-diagnostics
